draft-ietf-scitt-architecture.md

```diff
@@ -388,31 +388,35 @@ Multi-tenant support can be enabled through the use of identifiers in the `iss`
 
 Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Append-only Log.
 
-Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is eligible for registration.
-Registration Policies MUST be made transparent and available to all Relying Parties of the Transparency Service by registering them as Signed Statements on the Append-only Log, and distributing associated Receipts.
-Distribution of Receipts is out of scope for this document.```
+Transparency Services MUST maintain Registration Policies.
 
-This specification leaves implementation, encoding and documentation of Registration Policies to the operator of the Transparency Service.
+Transparency Services MUST also maintain a list of trust anchors used to authenticate Issuers, which MAY be included in a registration policy statement.
+For instance, a trust anchor could be an X.509 root certificate, the discovery URL of an OpenID Connect identity provider, or any other COSE compatible PKI trust anchor.
+
+Registration Policies and trust anchors MUST be made transparent and available to all Relying Parties of the Transparency Service by registering them as Signed Statements on the Append-only Log, and distributing the associated Receipts.
+
+This specification leaves implementation, encoding and documentation of Registration Policies and trust anchors to the operator of the Transparency Service.
 
 #### Mandatory Registration Checks
 
-Transparency Services MUST, at a minimum, perform the following checks before registering a Signed Statement:
+During registration, a Transparency Service MUST, at a minimum, authenticate the Issuer of the Signed Statement by validating the COSE signature and checking the identity of the issuer against one of its currently configured trust anchors, using the `x5t` (34), `x5chain`(33) or `kid`(4) protected headers of the Signed Statement as hints.
+For instance, in order to authenticate X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
+
+The Transparency Service MUST evaluate the Registration Policy that was most recently added to the Append-only Log.
```

Comment:
@robinbryce if you are concerned about the lack of prescribed window for applying policy updates this could be weakened
```suggestion
The Transparency Service MUST evaluate the most recently registered Registration Policy for which a Receipt has been distributed.
```
This allows more asynchronous registration policy update, the TS can withhold the Receipt until it is ready to enable the new policy

Reply:
FAOD: I'm essentially ok with this PR for what that is worth. If we get something like ad suggests, which I belive avoids all the bootstrappy and ordering implications, I think I'm starting to like registration policies. But as long as they block registration, I'm anticipating many "well intentioned" and also "good" implementations wont benefit much from them.